NDB.Covid19/NDB.Covid19/NDB.Covid19.csproj

```diff
@@ -3,6 +3,9 @@
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
     <Configurations>Debug;Release;UNIT_TEST</Configurations>
+    <RestoureSources>
+        $(MSBuildThisFileDirectory);
```

Comment:
Seem to be not resolved automatically without explicitly adding nuget source in VS.

Reply:
Been through various iterations, and it now seems to be working both for me locally and in the pipeline